src/redux/selectors/ao/get_ao_params.js

```diff
@@ -0,0 +1,8 @@
+import _get from 'lodash/get'
+import { REDUCER_PATHS } from '../../config'
+
+const path = REDUCER_PATHS.AOS
+
+export default (state) => {
+  return _get(state, `${path}.aoParams`, [])
+}
```

Comment:
to avoid passing diff reference every time in case aoParams state is undefined
```suggestion

const EMPTY_ARR = []

export default (state) => {
  return _get(state, `${path}.aoParams`, EMPTY_ARR)
}
```